src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -55,13 +54,11 @@ Azure | [Dsv4-series VMs](https://learn.microsoft.com/azure/virtual-machines/dv4
 
 ## Step 3. Select the region(s)
```

Comment:
It might be good to structure this section as a numbered list, where 1. is "Select regions", and 2. is "Select number of nodes". The structure would then mirror Step 5. Configure cluster capacity, where you also have multiple steps.